pkg/vpcmodel/explainabilityPrint.go

```diff
@@ -118,6 +120,8 @@ func explainMissingCrossVpcRouter(src, dst string, connQuery *connection.Set) st
 // the connection is blocked and only part of the path is printed then 2 is printed so that the relevant information
 // is provided regardless of where the is blocking
 // 4 is printed only in detailed mode
+//
+//nolint:gocyclo // better not split into two function
```

Comment:
Consider breaking this function to sub-functions.
It got pretty huge and rather hard to follow.
e.g., put all code under `if verbose` in a different function